cabal-install/src/Distribution/Client/Config.hs

```diff
@@ -1314,6 +1315,19 @@ configFieldDescriptions src =
             ParseArgs
        ]
   where
+    toSavedConfig
+      :: (FieldDescr a -> FieldDescr SavedConfig)
+      -- Lifting function.
```

Comment:
NB: This function isn't rendered in Haddocks so these comments can't be Haddock comments.